### PR TITLE
Improved spelling and upgraded to Samply design / colour scheme

### PR DESCRIPTION
**Pull request by dolchchr**
Improvements done:
- English spelling improved (thanks to Cecilia et. al.)
- Official Samply Logo added
- Design and colour scheme adapted to the official Samply Logo

### CODE REVIEW

**Review by alexanderkiel — COMMENTED** [review]
Why did you add those transitions?

**Review by alexanderkiel — COMMENTED** [review]
Can you please use `<p>` for paragraphs and remove all `<br>`'s.

**Comment by dolchchr** [comment]
> Why did you add those transitions?

for a smoother hover effect on weblinks

**Comment by alexanderkiel** [comment]
> > Why did you add those transitions?
> 
> for a smoother hover effect on weblinks

No I mean the transitions on the title header.

**Comment by dolchchr** [comment]
> > > Why did you add those transitions?
> > 
> > 
> > for a smoother hover effect on weblinks
> 
> No I mean the transitions on the title header.

I was not sure about bg-colour and with this effect it was easy to compare difference colours. Finally i choose black over white (over all others) but i liked the effect and didnt remove it. small easteregg.

**Comment by alexanderkiel** [comment]
> > > > Why did you add those transitions?
> > > 
> > > 
> > > for a smoother hover effect on weblinks
> > 
> > 
> > No I mean the transitions on the title header.
> 
> I was not sure about bg-colour and with this effect it was easy to compare difference colours. Finally i choose black over white (over all others) but i liked the effect and didnt remove it. small easteregg.

Or simply a [Duck](https://blog.codinghorror.com/new-programming-jargon/). :-)

**Comment by dolchchr** [comment]
> > > > > Why did you add those transitions?
> > > > 
> > > > 
> > > > for a smoother hover effect on weblinks
> > > 
> > > 
> > > No I mean the transitions on the title header.
> > 
> > 
> > I was not sure about bg-colour and with this effect it was easy to compare difference colours. Finally i choose black over white (over all others) but i liked the effect and didnt remove it. small easteregg.
> 
> Or simply a [Duck](https://blog.codinghorror.com/new-programming-jargon/). :-)

By the way: Do you also prefer black over white? I think its the best contrast to the green/gray logo, but looks a little bit "unusual". White fits good to the site, but contrast to the green is (too) low in my opinion.

**Comment by alexanderkiel** [comment]
> > > > > > Why did you add those transitions?
> > > > > 
> > > > > 
> > > > > for a smoother hover effect on weblinks
> > > > 
> > > > 
> > > > No I mean the transitions on the title header.
> > > 
> > > 
> > > I was not sure about bg-colour and with this effect it was easy to compare difference colours. Finally i choose black over white (over all others) but i liked the effect and didnt remove it. small easteregg.
> > 
> > 
> > Or simply a [Duck](https://blog.codinghorror.com/new-programming-jargon/). :-)
> 
> By the way: Do you also prefer black over white? I think its the best contrast to the green/gray logo, but looks a little bit "unusual". White fits good to the site, but contrast to the green is (too) low in my opinion.

I wouldn't use boxes for headlines at all. But I think, we can improve in the styling in a next PR.

**Review by johnlinde — APPROVED** [review]

**Review by TasDeniz — APPROVED** [review]

**Review by noemide — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -25,185 +27,109 @@
...
+ -o-transition: all 0.75s ease;
+ transition: all 0.75s ease;
 }
 </style>
 </head>
 
 <body>
-<h1>The Samply manifest</h1>
-<br />
+<h1><img src="./Samply.png" alt="Samply"><b style="vertical-align:super"> manifest<b></h1>
```
Comment (alexanderkiel) [inline index.html]: Please close the `b` tag here.

File: index.html [hunk 2]
```diff
@@ -25,185 +27,109 @@
...
+<p>We keep external dependencies as low as possible. External libraries have to be suggested to the Samply committee and agreed by the affected component maintainers before use.</p>
+<p>Each modification should be communicated to the component maintainer. Other involved community members should also be informed promptly.</p>
+<p>Whenever a modification is made to interfaces, we will perform a system test.</p>
+<p>We use a Samply license header for our software. Embedding this header is a requirement for each Samply component.</p>
+<p>For each release, we ensure compatibility is maintained between connected Samply components (for combinations, check the releases tabs or the docker compose files of the deployments at <a href="https://github.com/samply">GitHub/Samply</a>). If necessary, we modify affected Samply components to ensure compatibility.</p>
+<p>For very good reasons, it is possible to deviate from the rules of this criteria catalogue. Please inform affected community members and the Samply committee about any deviations promptly.</p>
 
 </body>
 </html>
```
Comment (alexanderkiel) [inline index.html]: A newline at the end of file would be good.

**Review by alexanderkiel — CHANGES_REQUESTED** [review]

**Review by BjoernKrollGba — APPROVED** [review]

**Review by alexanderkiel — APPROVED** [review]

**Pull request merged**